packages/dashboard/src/lib/framework/extension-api/types/form-components.ts

```diff
@@ -10,7 +10,15 @@ import { CustomFormComponentInputProps } from '../../form-engine/custom-form-com
  * @since 3.4.0
  */
 export interface DashboardCustomFormComponent {
+    /**
+     * @description
+     * A unique identifier for the custom form component.
```

Comment:
Add "it is a good practice to name-space these IDs to avoid naming collisions, for example "my-plugin.markdown-editor".